# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: a1ab26abf3eb
# Full Hash: a1ab26abf3eb645e202d69dffea86398a0ea4563
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-09-30 21:45:29
# Regressor Bug: 1667192
# File Overlap Count: 6
# Description:
#   Bug 1667192 - patch 1.1 - Run `mach vendor rust` to update in-tree mapped_hyph source. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D91409
# ==============================================================================

diff -r 66b86f8b6682 -r a1ab26abf3eb .cargo/config.in
--- a/.cargo/config.in	Wed Sep 30 12:51:17 2020 +0000
+++ b/.cargo/config.in	Wed Sep 30 12:51:54 2020 +0000
@@ -40,7 +40,7 @@
 [source."https://github.com/jfkthame/mapped_hyph.git"]
 git = "https://github.com/jfkthame/mapped_hyph.git"
 replace-with = "vendored-sources"
-tag = "v0.3.0"
+tag = "v0.4.0"
 
 [source."https://github.com/hsivonen/packed_simd"]
 git = "https://github.com/hsivonen/packed_simd"